eras/mary/impl/src/Cardano/Ledger/Mary/TxCert.hs

```diff
@@ -5,13 +5,17 @@ module Cardano.Ledger.Mary.TxCert () where
 
 import Cardano.Ledger.Crypto (Crypto, StandardCrypto)
 import Cardano.Ledger.Mary.Era (MaryEra)
-import Cardano.Ledger.Shelley.TxCert (EraTxCert (..), ShelleyEraTxCert (..), ShelleyTxCert (..))
+import Cardano.Ledger.Shelley.TxCert
```

Comment:
non-Core open import?

Reply:
There are three imports in this module. Only one of them is open, which means that everything that is not imported explicitly comes from `Cardano.Ledger.Shelley.TxCert` (aside from Prelude of course)

That being said, I know how much you like those explicit imports, so I'll add it back.

Reply:
It's true I don't like them. but more I was just hoping for a consistent rule about when we use them. I can be flexible too :)